docs/io.cozy.files.md

```diff
@@ -168,6 +168,63 @@ The `io.cozy.files` doctype has [the standard `cozyMetadata`](https://docs.cozy.
 }
 ```
 
+### Shortcuts
+
+Shortcuts (or hyperlinks) are a special type of files. They are served
+with the `application/shortcut` mime-type, they have a `.url` extension, and
+respect the [URL file
+format](http://www.lyberty.com/encyc/articles/tech/dot_url_format_-_an_unofficial_guide.html).
+
+They can have `metadata`:
+
+- `sharing-status` will be set if the link can be used to open a sharing. 2
+  values are possible: `new` and `seen`, depending on whether the link
+  has been opened.
+  has been opened or not.
+- `target-cozy` is set to the cozy instance when relevant (mostly sharing
```

Comment:
should we have a root `target: { … }` for all this instead of `metadata: { target-* }` properties ?

Reply:
If so, should we try to mimic an actual file in this `target` ? Like : 

```
{
  "_id": "629fb233be550a21174ac8e19f0043af",
  "_rev": "1-61c7804bdb4f9f8dae5a363cb9a30dd8",
  "type": "file",
  "name": "sunset.jpg.url",
  "dir_id": "629fb233be550a21174ac8e19f003e4a",
  "trashed": false,
  "md5sum": "vfEMDpJShs8QeIlsDmw9VA==",
  "created_at": "2020-02-10T20:38:04Z",
  "updated_at": "2020-02-10T20:38:04Z",
  "tags": [],
  "target": {
    "mime": "image/jpg",
    "_type": "io.cozy.files",
    "cozyMetadata": {
      "createdOn": "https://alice.cozy.example/",
     }
  },
  "metadata": {
    "sharing-status": "new",
  },
  "size": 62,
  "executable": false,
  "class": "shortcut",
  "mime": "application/shortcut",
  "cozyMetadata": {
    "doctypeVersion": 1,
    "metadataVersion": 1,
    "createdAt": "2020-02-10T20:38:04Z",
    "createdOn": "https://bob.cozy.example/",
    "updatedAt": "2020-02-10T20:38:04Z",
    "uploadedAt": "2020-02-10T20:38:04Z",
    "uploadedOn": "https://bob.cozy.example/"
  }
}
```

Reply:
We could have: 

```json
"metadata": {
  "target": {
    "mime": "image/jpg",
    "_type": "io.cozy.files",
    "cozyMetadata": {
      "createdOn": "https://alice.cozy.example/",
     }
  },
  "sharing-status": "new"
}
```

But I would prefer to keep those attributes in the metadata. We need to allow web apps to give those fields, and metadata is good place for that. For example, for a link to `https://alice-calendar.cozy.example/#/123456`, the stack won't know how to fill the mime-type (is it an event or a calendar?).

Reply:
And I don't think that `cozyMetadata.createdOn` is a good way to indicate the target cozy. Technically, the destination file can have been created somewhere else (like on dave cozy instance). It's just that Bob have a link that goes to Alice instance, but it doesn't say anything about the past of this target.

Reply:
Ok for me in metadata if you prefer. Having a structured `target` will let us fill it with  more later, and still reuse some functions we already have `target`having the same structure as a standard file)

@Crash-- what do you think?

Reply:
> And I don't think that `cozyMetadata.createdOn` is a good way to indicate the target cozy

:+1:
`ownedBy` ?

Reply:
I like the idea of having something as close as possible to regular file. 

Also, having a `scope` target in the metadata object is more readable & understandable I think. Also easier to delete if needed